qute.jdt/com.redhat.qute.jdt/src/main/java/com/redhat/qute/jdt/utils/IJDTUtils.java

```diff
@@ -85,5 +86,26 @@ public interface IJDTUtils {
 
 	Location toLocation(IJavaElement element) throws JavaModelException;
 
-	//String getJavadoc(IMember member, DocumentFormat documentFormat) throws JavaModelException;
+	/**
+	 * Returns the JDT text edit as an LSP text edit
+	 *
+	 * @param compulationUnit the compilation unit of the file that was edited
+	 * @param jdtTextEdit     the JDT text edit
+	 * @return the JDT text edit as an LSP text edit
+	 */
+	TextDocumentEdit convertEdit(ICompilationUnit compulationUnit, org.eclipse.text.edits.TextEdit jdtTextEdit);

```

Comment:
After seeing the code of TextEditConverter of JDT LS, I think you can copy/paste the TextEditConverter and adapt it to use it with our own IJDTUtils. It will avoid for @jeffmaury to implement this method.